src/textures/VideoTexture.js

```diff
@@ -40,8 +27,35 @@ class VideoTexture extends Texture {
 	update() {
 
 		const video = this.image;
+
+		if ( video == null ) {
+
+			this._callbackSource = null;
+
+			return;
+
+		}
+
 		const hasVideoFrameCallback = 'requestVideoFrameCallback' in video;
 
+		if ( hasVideoFrameCallback && video !== this._callbackSource ) {
```

Comment:
You should stop the previous callback loop when creating a new one.

Reply:
Yep, this is problem.

We should store and callbackId and source reference, that add already 2 extra field to instance, what is undesirable

Reply:
Another way to solve would be just to check in `update` function whether the callback source and the video are still the same, and request callback only if they still are.

Reply:
This is bad. 
requirestFrameCallback is desynced. Some video has 30 fps, and will update one of 2 of frames.
Some require tick 90+ FPS, and there are a monitors that tick 240 FPS.

update is ticks with render FPS, and you can't requiest video callback from update each call.

Reply:
The check can be made in the start of `update`, so if the video changes the old callback would return before setting `needsUpdate` to `true`.

> update is ticks with render FPS, and you can't requiest video callback from update each call.

But you do currently?